packages/grafana-data/src/utils/namedColorsPalette.ts

```diff
@@ -49,6 +49,8 @@ export type ColorDefinition = {
 };
 
 let colorsPaletteInstance: Map<Hue, ColorDefinition[]>;
+let colorsMap: Record<Color, string> | undefined;
+let colorsMapTheme: GrafanaTheme | undefined;
```

Comment:
nit: to me these global lets indicates we should have a wrapping function or class instead.

Reply:
Interesting, why? These are module-scoped, not sure I understand the benefit of introducing another abstraction over those.

@torkelo one thing I would consider ad some point is to start namespacing the utilities. Currently we expose all of these as top level exports. Think it would possibliy be easier for the API consumers to import namespaces first and then explore the APIs. But this is fairly big change I think, something that we can think about for major version